### ISSUE TITLE
[Feature Request][NOWEB] - Set the session name on personal Whatsapp on your cell phone.

### ISSUE THREAD

**Issue by gsoftdobrasil** [allburov/whatsapp-http-api#326]
I would like to be able to set the session name in Whatsapp after making the connection. Today it is automatically set to Google Chrome (Ubuntu) on personal Whatsapp. On Whatsapp Business I can give the name myself. Can't this resource be the same?

[![patron:PLUS](https://img.shields.io/badge/patron-PLUS-a0e6ba)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
Yes, we had such request in the past, will look at this for sure!
https://github.com/devlikeapro/whatsapp-http-api/issues/220


[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
